tests/integration/test_projectconfigs.py

```diff
@@ -294,34 +294,31 @@ def test_unparsable_project_config(buffer_config, mini_sentry, relay):
         }
     } == data
 
+    def assert_clear_test_failures():
+        try:
+            assert {str(e) for _, e in mini_sentry.test_failures} == {
+                f"Relay sent us event: error fetching project state {public_key}: invalid type: integer `99`, expected a string",
+            }
+        finally:
+            mini_sentry.test_failures.clear()
```

Comment:
I copied it and wasn't sure myself. I **think** it is here in case there are other failures, they would already raise an error from the assert and keeping the test failures it would show them again when exiting the test (through the normal assertion behaviour).